games/game_witcher3.py

```diff
@@ -11,20 +15,32 @@ class Witcher3Game(BasicGame):
 
     Name = "Witcher 3 Support Plugin"
     Author = "Holt59"
-    Version = "1.0.0a"
-    Description = "The Description Of The Dead"
+    Version = "1.0.1"
+    Description = "The Description Of The Dead| Updated to work with GOG by Luca/EzioTheDeadPoet"
 
     GameName = "The Witcher 3: Wild Hunt"
     GameShortName = "witcher3"
     GaneNexusHame = "witcher3"
     GameNexusId = 952
     GameSteamId = 292030
     GameBinary = "bin/x64/witcher3.exe"
-    GameDataPath = "Mods"
+    GameDataPath = ""
```

Comment:
Top level mods are likely to not work due to being dlls. Rather that having something that doesn't work half the time it's better to offer a more limited feature that works all the time. So leave /mods folder specification.

Reply:
I agree with Al here. Furthermore, people are using this plugin, so changing it now would break their setup.